airflow/providers/common/sql/CHANGELOG.rst

```diff
@@ -24,6 +24,21 @@
 Changelog
 ---------
 
+1.1.1
+.....
+
+Bug Fixes
+~~~~~~~~~
+
+* ``Fix placeholders in 'TrinoHook', 'PrestoHook', 'SqliteHook' (#25939)``
+* ``Fix (and test) SQLTableCheckOperator on postgresql (#25821)``
```

Comment:
To avoid this bug for future postgres releases don't we need to set 
`apache-airflow-providers-common-sql>=1.1.1` in postgres provider yaml and include release for postgres provider as well?
https://github.com/apache/airflow/blob/a2db8fcb7df1a266e82e17b937c9c1cf01a16a42/airflow/providers/postgres/provider.yaml#L43

Reply:
Ah you already fixed it :) 
resolving this

Reply:
Yes. We should bump all the providers that use common.sql and add >=1.2.0 for all of them